oxytcmri/tests/unit/domain/use_cases/test_segment_dti_abnormal_values.py

```
@@ -1,20 +1,24 @@
-from typing import List
+from dataclasses import dataclass
```

Ruff (0.11.9)

`dataclasses.dataclass` imported but unused

Remove unused import: `dataclasses.dataclass`

(F401)